db/dcrpg/queries.go

```diff
@@ -309,8 +314,9 @@ func InsertTickets(db *sql.DB, dbTxns []*dbtypes.Tx, txDbIDs []uint64, checked,
 //
 // Outputs are slices of DB row IDs for the votes and misses, and an error.
```

Comment:
What is `votesMilestones` used for, and is it modified?

Reply:
This is has the `deployment` map found in the blockchain info map. It hold the updated version of the blockchain info.

Reply:
I mean please describe in the docs